zebra-state/src/error.rs

```diff
@@ -43,7 +43,7 @@ pub type BoxError = Box<dyn std::error::Error + Send + Sync + 'static>;
 
 /// An error describing the reason a block could not be committed to the state.
 #[derive(Debug, Error, PartialEq, Eq)]
-#[error("block is not contextually valid")]
+#[error("block is not contextually valid: {}", .0)]
```

Comment:
Would you be able to provide a reference to the  .0 part?. I haven't seen that syntax yet, and I didn't find any docs mentioning it.

Reply:
I think it's documented in the third bullet point here: https://docs.rs/thiserror/latest/thiserror/#details

In this case I think it could be just `{0}`